File_Format_Library/FileFormats/Texture/BTI.cs

```diff
@@ -47,53 +49,58 @@ private int RoundHeight(int height, int BlockHeight)
             return height + ((BlockHeight - (height % BlockHeight)) % BlockHeight);
         }
 
+        private void Read(System.IO.Stream stream)
+        {
+            
+        }
+
         public void Load(System.IO.Stream stream)
         {
             //Set this if you want to save the file format
             CanSave = true;
+            CanEdit = false;
+
+            ImageKey = "Texture";
+            SelectedImageKey = "Texture";
+            Text = FileName;
 
             //You can add a FileReader with Toolbox.Library.IO namespace
             using (var reader = new FileReader(stream))
             {
-                Texture tex = new Texture();
-                tex.CanEdit = false;
+                CanEdit = false;
 
                 reader.SetByteOrder(true);
 
                 //Turn this format into a common format used by this tool
                 byte texFormat = reader.ReadByte();
-                tex.Format = Decode_Gamecube.ToGenericFormat((Decode_Gamecube.TextureFormats)texFormat);
+                Format = Decode_Gamecube.ToGenericFormat((Decode_Gamecube.TextureFormats)texFormat);
 
                 _ = reader.ReadByte(); // enable alpha
-                tex.Width = reader.ReadUInt16();
-                tex.Height = reader.ReadUInt16();
+                Width = reader.ReadUInt16();
+                Height = reader.ReadUInt16();
                 _ = reader.ReadByte(); // wrap s
```

Comment:
Adjust the variable names as  appveyor doesn't like those. If you don't use any, you can just leave out the _ = part.

Reply:
Alright, I'll do so now but dont forget the BTI class inherits 
STGenericTexture, IFileFormat, ISingleTextureIconLoader those so it is technically correct right?
otherwise I don't know how to assign to the width and height